Extension/src/LanguageServer/configurations.ts

```diff
@@ -103,6 +102,8 @@ export interface Browse {
 export interface KnownCompiler {
     path: string;
     isC: boolean;
+    isTrusted: boolean;
```

Comment:
Is this supposed to be used in the TypeScript?

Reply:
...same as isCl. I'm not sure why it's being returned if they're not used.

Reply:
Yes, it is, it is necessary to parse the properties that are sent in the json object received on the typescript side.

Reply:
But why do the properties exist on the object in the first place?

Reply:
I don't think I understand, are you asking why we keep track of whether a compiler is trusted or not?